### PR TITLE
find sign by signName endpoint

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

**Comment by yCobanoglu** [comment]
[Create interface for signs](https://trello.com/c/Ss2PYZ15/492-create-interface-for-signs)

File: ebl/signs/web/bootstrap.py [hunk 1]
```diff
@@ -2,8 +2,11 @@
...
 from ebl.context import Context
 from ebl.signs.web.sign_search import SignsSearch
+from ebl.signs.web.signs import SignsResource
 
 
 def create_signs_routes(api: falcon.API, context: Context):
     signs_search = SignsSearch(context.sign_repository)
+    signs = SignsResource(context.sign_repository)
     api.add_route("/signs", signs_search)
+    api.add_route("/signs/{object_id}", signs)
```
Comment (jlaasonen) [inline ebl/signs/web/bootstrap.py:12]: Why is the parameter called "object_id" when it is not `ObjectId`?

**Review by jlaasonen — CHANGES_REQUESTED** [review]

File: ebl/signs/web/bootstrap.py (hunk 1 above)
Comment (yCobanoglu) [reply ebl/signs/web/bootstrap.py:12]: i used it so it is similar to the words route. I will change it.

**Review by jlaasonen — APPROVED** [review]

File: ebl/signs/web/bootstrap.py (hunk 1 above)
Comment (jlaasonen) [reply ebl/signs/web/bootstrap.py]: Words route is the first route implemented. The name is probably a leftover from a time when `ObjectId` was used and should be updated.

**Pull request merged**